app/services/host_fact_importer.rb

```diff
@@ -31,24 +27,24 @@ def import_facts(facts, source_proxy = nil)
       facts_importer.import!
     end
 
-    skiping_orchestration do
+    skipping_orchestration do
       host.save(:validate => false)
     end
 
     parse_facts(facts, type, source_proxy)
   end
 
   def parse_facts(facts, type, source_proxy)
-    skiping_orchestration do
+    skipping_orchestration do
       host.parse_facts facts, type, source_proxy
     end
   end
 
   private
 
-  def skiping_orchestration(&block)
+  def skipping_orchestration(&block)
     if host.is_a?(Host::Managed)
-      host.without_orchestration_if(skip_orchestration?, &block)
+      host.without_orchestration_if(true, &block)
```

Comment:
That's the default value, so no need to actually pass around `&block`:
```suggestion
      host.without_orchestration_if
        yield
      end
```
Also, considering that this is the only consumer of this method, do we actually need the parameter or should it just be renamed to `.without_orchestration` and the logic of whether or not it needs skipping will stay with the caller?

Reply:
I guess it's fair :+1: